analyzers/rspec/vbnet/S4545_vb.net.html

```diff
@@ -0,0 +1,23 @@
+<p>The <code>DebuggerDisplayAttribute</code> is used to determine how an object is displayed in the debugger window.</p>
+<p>The <code>DebuggerDisplayAttribute</code> constructor takes a single argument: the string to be displayed in the value column for instances of the
+type. Any text within curly braces is evaluated as the name of a field, property, or method.</p>
+<p>Naming a non-existent field, property or method between curly braces will result in a CS0103 error in the debug window when debugging objects.
```

Comment:
C# and VB do not share compilation errors and warnings
```suggestion
<p>Naming a non-existent field, property or method between curly braces will result in a BC30451 error in the debug window when debugging objects.
```

Reply:
Ok, I run it incorrectly. 
It should be fixed now, targeting the content of https://github.com/SonarSource/rspec/pull/1571.